src/utils/configValidators/boolean.js

```diff
@@ -12,7 +12,7 @@ governing permissions and limitations under the License.
 
 import isBoolean from "../isBoolean";
 
-export default (config, key, currentValue) => {
+export default (key, currentValue) => {
   return isBoolean(currentValue)
```

Comment:
I believe this should return an empty string if the value is `undefined`. I'm not sure if you wanted to do that as part of this same PR.